lib/test-util-macros/src/lib.rs

```diff
@@ -0,0 +1,143 @@
+// The MIT License (MIT)
```

Comment:
Does this have implications for the license on the main Vector package?

Reply:
Good question. Unfortunately, I do not have an answer. I placed license to file because `src/stream.rs` had the same solution: https://github.com/timberio/vector/blob/8e7a3674ffca11b13a3a87dcf69d7254f2083e98/src/stream.rs#L1-L23